### ISSUE TITLE
Flaky-test: testManagedLedgerBookieClientStats(org.apache.pulsar.broker.stats.PrometheusMetricsTest)  fails very often on CI

### ISSUE THREAD

**Issue by eolivelli** [apache/pulsar#8374]
I am seeing very often this error on GitHub Actions.
I am going to send a fix if there is no current work about this issue

```
Error:  testManagedLedgerBookieClientStats(org.apache.pulsar.broker.stats.PrometheusMetricsTest)  Time elapsed: 0.402 s  <<< FAILURE!
java.lang.NumberFormatException: For input string: "-Inf"
	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
	at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.lang.Double.parseDouble(Double.java:538)
	at java.lang.Double.valueOf(Double.java:502)
	at org.apache.pulsar.broker.stats.PrometheusMetricsTest.lambda$parseMetrics$8(PrometheusMetricsTest.java:572)
	at java.lang.Iterable.forEach(Iterable.java:75)
	at org.apache.pulsar.broker.stats.PrometheusMetricsTest.parseMetrics(PrometheusMetricsTest.java:562)
	at org.apache.pulsar.broker.stats.PrometheusMetricsTest.testManagedLedgerBookieClientStats(PrometheusMetricsTest.java:518)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

Error:  testManagedLedgerCacheStats(org.apache.pulsar.broker.stats.PrometheusMetricsTest)  Time elapsed: 0.247 s  <<< FAILURE!
java.lang.NumberFormatException: For input string: "-Inf"
	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2043)
	at sun.misc.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
	at java.lang.Double.parseDouble(Double.java:538)
	at java.lang.Double.valueOf(Double.java:502)
```

**Comment by eolivelli** [comment]
This is probably a problem introduced with #6814 fix

**Comment by eolivelli** [comment]
fix is #8379
